services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/proxy.py

```diff
@@ -52,7 +52,7 @@ def get_dynamic_proxy_spec(
             "swarm_stack_name": dynamic_sidecar_settings.SWARM_STACK_NAME,
             "traefik.docker.network": swarm_network_name,
             "traefik.enable": "true",
-            f"traefik.http.middlewares.{scheduler_data.proxy_service_name}-security-headers.headers.customresponseheaders.Content-Security-Policy": f"frame-ancestors {scheduler_data.request_dns}",
+            f"traefik.http.middlewares.{scheduler_data.proxy_service_name}-security-headers.headers.customresponseheaders.Content-Security-Policy": f"frame-ancestors {scheduler_data.request_dns} {scheduler_data.node_uuid}.services.{scheduler_data.request_dns}",
```

Comment:
Don't you need commas between these two options? I see other entries of traefik that requires that

Reply:
Q: ``scheduler_data.node_uuid}.services.{scheduler_data.request_dns}`` is the entrypoint of a dynamic services, right? I see that the project id is not used for that. Does this assume that nodes have universal identifiers? i.e. two projects cannot include a node with the same NodeID as dynamic service ..

Reply:
Yes, that is exactly the entrypoint and we are saying to CSP that it can start iframes, to allow for iframe in iframes.

Yes, that is the same assumption as the legacy services!

Reply:
@GitHK @sanderegg OK, just for the record: 
it would be good to identify dynamic services with something more than the node UUID (e.g. project_uuid/node_uuid)

We would like to guarantee the node-uuid uniquenes only within the scope of its project. That would help us a lot on:
- simplifies duplication of projects
- simplifies caching in metamodling iterations
- ...

See  ITISFoundation/osparc-simcore#2133 for more details